client/Windows/wf_defaults.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FREERDP_CLIENT_WIN_DEFAULTS_H
```

Comment:
You might add the copyright and license header with your name here ;)
see other files for examples